scripts/gn/templates/auto_template.yaml

```diff
@@ -0,0 +1,319 @@
+inputs:
+
+    inputs_root: "."
+
+    atx_files:  [ ../../inputData/products/igs20.atx                    ] # required
+    egm_files:  [ ../../inputData/products/tables/EGM2008.gfc           ] # Earth gravity model coefficients file
+    igrf_files: [ ../../inputData/products/tables/igrf13coeffs.txt             ]
+    erp_files:  [ ../../inputData/products/tables/finals.data.iau2000.txt  ]
+    planetary_ephemeris_files: [ ../../inputData/products/tables/DE436.1950.2050  ]
+
+    troposphere:
+        gpt2grid_files: [ ../../inputData/products/gpt_25.grd ]
+
+    tides:
+        ocean_tide_loading_blq_files: [ ../../inputData/products/OLOAD_GO.BLQ                ] # required if ocean loading is applied
+        ocean_tide_potential_files:   [ ../../inputData/products/tables/fes2014b_Cnm-Snm.dat ]
+
+    snx_files:  [ "./downloads/*.SNX" ]
+
+    satellite_data:
+        clk_files: [ "./downloads/*.CLK"     ]
+        bsx_files: [ "./downloads/*.BSX"     ]
+        sp3_files: [ "./downloads/*.SP3"     ]
+
+    gnss_observations:
+        gnss_observations_root: ./data/
+        rnx_inputs:
+            - "*.rnx"
+
+
+outputs:
+
+    metadata:
+        config_description:    "187369_v3"
+
+    outputs_root:              ./outputs/
+
+    # gpx:
```

Comment:
It would be good to have gpx turned on by default - it's an easier way for users to grab results after the run. The TRACE is ok, but it's a big file and it can be hard to find the details you want
I think this will be my last comment on this (happy for it to go in after this).